tests/PromiseAdapter/PromiseAdapterInterface.php

```diff
@@ -6,6 +6,9 @@
 
 interface PromiseAdapterInterface
 {
+    /**
+     * @phpstan-ignore-next-line
```

Comment:
Is this still needed? Should probably be `@return PromiseInterface<T>`, no?

Reply:
Yes, but see comment on `PromiseFulfilledTestTrait`. Would like to have it in, but IMHO not worth it right now.